webstruct/tests/test_add_pattern_features.py

```diff
@@ -0,0 +1,40 @@
+from webstruct.features.global_features import _add_pattern_features
```

Comment:
Would you mind testing Pattern class instead? It should be an easy change, but this way we'd be testing public API